net/unbound/files/unbound-odhcpd.defaults

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+[ "$(uci get dhcp.odhcpd.maindhcp)" = "1" ] || exit 0
+[ "$(uci get unbound.ub_main.dhcp_link)" = "none" ] || exit 0
+
+[ -x /usr/sbin/unbound-control ] && uci set unbound.ub_main.unbound_control='1'
+
+uci batch <<EOF
+set unbound.ub_main.add_local_fqdn='1'
+set unbound.ub_main.add_wan_fqdn='1'
```

Comment:
This one is certainly optional. It overrides the WAN DNS which may or may not have a public definition by your ISP.